git/git.go

```diff
@@ -22,3 +23,37 @@ func Clone(repo, destination string) error {
 	}
 	return nil
 }
+
+// Fetch fetches a pull request into a branch named "_pr"
+func Fetch(prNumber string) error {
+	cmd := execCommand("git", "fetch", "origin", "pull/"+prNumber+"/head:_pr")
+	cmd.Stdout = os.Stdout
+
+	err := cmd.Start()
+	if err != nil {
+		return fmt.Errorf("starting git fet command: %v", err)
+	}
+
+	err = cmd.Wait()
+	if err != nil {
+		return fmt.Errorf("Fetching pull request: %v", err)
+	}
+	return nil
+}
+
+// Merge merges changes on the "_pr" branch
+func Merge() error {
```

Comment:
Oh, it merges _pr? Yeah that's too magical. Merge should just merge what you tell it to merge, rather than assuming that some stuff happened before it was run.